.github/pull_request_template.md

```diff
@@ -1,21 +1,25 @@
 # Description
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Please include a summary of the change and which issue is fixed.
+Please also include relevant motivation and context.
+List any dependencies that are required for this change.
 
-Fixes [#8](https://usxtech.atlassian.net/browse/CLOUD-8)
+Fixes [#8](https://usxpress.atlassian.net/browse/CLOUD-8)
```

Comment:
I don't have access to this link

Reply:
Because it doesn't exist and is a dummy.